docs/platforms/python/tracing/instrumentation/custom-instrumentation/index.mdx

```diff
@@ -145,21 +173,21 @@ sentry_sdk.init(
 
 Now, whenever a function specified in `functions_to_trace` will be executed, a span will be created and attached as a child to the currently running span.
 
-<Alert level="warning" title="Important">
+<Alert title="Important" level="info">
 
 To enable performance monitoring for the functions specified in `functions_to_trace`, the SDK needs to load the function modules. Be aware, there may be code being executed in modules during module loading. To avoid this, use the method described above to trace your functions.
 
 </Alert>
```

Comment:
I feel like this page already has so much information. This block will just confuse people. We should either remove it, or if it is important, let's describe `functions_to_trace` in depth on a different page.

```suggestion
```